repo-issues/lib/data-source.js

```diff
@@ -13,10 +12,12 @@ const languageIconMap = {
 
 let showIssues = false
```

Comment:
these have default values but you always overwrite them in the `generateViewState`, it might be less confusing to not initialise them, what do you think?

Reply:
now that you point out yep, I use them for debugging but in the real thing they'll never be used: this is overriding them in any case